pages/common/import.md

```diff
@@ -0,0 +1,15 @@
+# import
+
+> Capture some or all of an X server screen and save the image to a file. Part of ImageMagick library
+
+- Capture the entire X server screen in the Postscript image format:
+
+`import -window root {{filename.postscript}}`
+
+- Capture contents of remote x server screen in the png image format:
+
+`import -window root -display {{remote.host}}:0.0 {{output.png}}`
```

Comment:
Why is the remote host in the format of `remote.host` ? I think its better to mention it as `remote_host` ?